logging/logging.go

```diff
@@ -6,6 +6,27 @@ import (
 
 type Logger = log15.Logger
 
+type Level = log15.Lvl
```

Comment:
All this bullshit is related to the fact that I am trying to avoid importing log15 all over the application so my goal is to simply import `github.com/lovelaced/glitzz/logging` everywhere. Hence the type aliases and wrapper methods.